squire_web/src/tournament/players/selected.rs

```diff
@@ -1,16 +1,22 @@
 use squire_sdk::{
-    players::{Player, PlayerId},
-    tournaments::Tournament,
+    players::{Player, PlayerId, Round},
+    tournaments::Tournament, model::{identifiers::{PlayerIdentifier, TypeId}, rounds::RoundId},
 };
 use yew::prelude::*;
 
 pub struct SelectedPlayer {
+    pub process: Callback<RoundId>,
     id: Option<PlayerId>,
+    round_id: Option<RoundId>,
```

Comment:
I see what you're doing here, and it makes total sense as a first draft. However, we also want to allow TOs to select a deck. So, instead of an `Option<RoundId>`, this should be an `Option<SelectedPlayerInfo>` and we define `SelectedPlayerInfo` as:
```rust
enum SelectedPlayerInfo {
    Round(RoundId),
    Deck(String),
}
```
We can then change the update, render, and callback logic to match.